src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -269,6 +273,17 @@ def for_schema_info(self, schema: SchemaInfo):
             principal_permissions[grant.principal].add(grant.action_type)
         return principal_permissions
 
+    # The reported ObjectType for grants doesn't necessary match the type of grants we asked for. This maps
+    # those that aren't themselves.
+    _grants_reported_as = {
+        # SHOW type: RESULT type
+        "SCHEMA": "DATABASE",
+        "CATALOG": "CATALOG$",
```

Comment:
Why the dollar sign?

Reply:
This table maps what we ask for in the `SHOW` statement to the object type as reported in the result-set. In the case of Catalog-scoped permissions it goes like this:
```python
spark.sql("show grants on catalog hive_metastore").show(truncate=False)
```
```
+--------------+----------+----------+---------+
|Principal     |ActionType|ObjectType|ObjectKey|
+--------------+----------+----------+---------+
|ajs_test_group|SELECT    |CATALOG$  |None     |
+--------------+----------+----------+---------+
```